distribution/docs/src/main/resources/_contents/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc

```diff
@@ -0,0 +1,48 @@
+.[[org.codice.alliance.nsili.endpoint]]NSILI Endpoint Configuration
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Max Number Returned Results
+|maxNumResults
+|Integer
+|Maximum number of results that will be returned.
+|500
+|true
+
+| Standing Query Update Rate
+| defaultUpdateFrequencySec
+| Integer
+| Default update rate for standing queries (seconds)
+| 60
+| true
+
+| Maximum Number Pending Results
+| maxPendingResults
+| Integer
+| Maximum Number of results to cache for each client.
+| 10000
+| true
+
+| Enabled Outgoing Validation
+| outgoingValidationEnabled
+| Boolean
+| Enabled or disabled outgoing DAG validation against mandatory attributes
+| 10000
```

Comment:
This default value should be a Boolean of `false`. The metatype.xml for the NsiliEndpoint has a typo on this @troymohl